packages/frontend-shared/src/gql-components/modals/SelectCloudProjectModal.vue

```diff
@@ -254,7 +255,26 @@ mutation SelectCloudProjectModal_SetProjectId( $projectId: String! ) {
           runs(first: 10) {
             nodes {
               id
-              ...RunCard
```

Comment:
Why was this change made?

Reply:
The `RunCard` fragment is defined in packages/app, I got an "unknown fragment" error from the gql codegen after moving this component into frontend-shared. Maybe there's a better way to define this fragment to make it reusable in both places.